src/test/java/teammates/test/cases/webapi/PublishFeedbackSessionActionTest.java

```diff
@@ -23,13 +30,90 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        ______TS("Typical case");
+
+        CourseAttributes course = typicalBundle.courses.get("typicalCourse1");
+        FeedbackSessionAttributes session = typicalBundle.feedbackSessions.get("session1InCourse1");
+
+        String[] params = {
+                Const.ParamsNames.COURSE_ID, course.getId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, session.getFeedbackSessionName(),
+        };
+
+        PublishFeedbackSessionAction publishFeedbackSessionAction = getAction(params);
+
+        JsonResult result = getJsonResult(publishFeedbackSessionAction);
+        FeedbackSessionData feedbackSessionData = (FeedbackSessionData) result.getOutput();
+
+        assertEquals(feedbackSessionData.getFeedbackSessionName(), session.getFeedbackSessionName());
+        assertEquals(feedbackSessionData.getPublishStatus(), FeedbackSessionPublishStatus.PUBLISHED);
```

Comment:
The data return by the sever is verified. However, you should add another assertion to verify the status of the feedback session in DB. (e.g. use `logic.getFeedbackSession()` and verify the publish status).

Reply:
Updated